contracts/crates/chainlink-solana/Cargo.toml

```diff
@@ -15,6 +15,9 @@ name = "chainlink_solana"
 default = []
 
 [dependencies]
-solana-program = "1.17.25"
-borsh = "0.9.1"
-borsh-derive = "0.9.1"
+solana-program = { version = "1.17.25" }
```

Comment:
Why the downgrade? We wanted to release a new library version with a newer solana program version (1.17 is EOL). Is it a CI issue?